src/call_hook.cpp

```diff
@@ -0,0 +1,62 @@
+#include "common.hpp"
+#include "call_hook.hpp"
+#include "memory/handle.hpp"
+
+namespace
+{
+	big::call_hook_memory g_call_hook_memory;

```

Comment:
Request the compiler to inline this and remove it from the namespace, this reeks of C. Maybe move it to the header or something.

Reply:
Wait, is this even globally referred to? I don't see anything else using it besides this class file. g_ refers to a global identifier that can be accessed by the entire program. The only function that refers to it is the call_hook constructor.